contrib/mhchem/mhchem.js

```diff
@@ -77,7 +77,7 @@ import katex from "katex";
   var chemParse = function (tokens, stateMachine) {
     // Recreate the argument string from KaTeX's array of tokens.
     var str = "";
-    var expectedLoc = tokens[tokens.length - 1].loc.start
+    var expectedLoc = tokens.length === 0 ? 0 : tokens[tokens.length - 1].loc.start
```

Comment:
This could be written more concisely as follows:

```suggestion
    var expectedLoc = tokens.length && tokens[tokens.length - 1].loc.start
```

What do you think?

Reply:
Done.